tdp/cli/commands/test_init.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2022 TOSIT.IO
+# SPDX-License-Identifier: Apache-2.0
+
+from click.testing import CliRunner
+from .init import init
+
+import os
+
+
+def test_tdp_init(collection_path, database_dsn, vars):
+    args = [
+        "--collection-path",
+        collection_path,
+        "--database-dsn",
+        database_dsn,
+        "--vars",
+        vars,
+    ]
+    runner = CliRunner()
+    result = runner.invoke(init, args)
+    assert result.exit_code == 0
+
+
+def test_tdp_init_db_is_created(collection_path, vars, tmp_path):
+    dp_path = tmp_path / "sqlite.db"
```

Comment:
What does `dp_path` stands for? Did you meant `db_path`?

Reply:
yes, typo